platform/javascript/js/libs/library_godot_os.js

```diff
@@ -282,6 +282,11 @@ const GodotOS = {
 	godot_js_os_shell_open: function (p_uri) {
 		window.open(GodotRuntime.parseString(p_uri), '_blank');
 	},
+
+	godot_js_os_hw_concurrency_get__sig: 'i',
+	godot_js_os_hw_concurrency_get: function () {
+		return navigator.hardwareConcurrency || 1;
```

Comment:
I wondered if `navigator.hardwareConcurrency` had something special, and it does!

> The browser may, however, choose to report a lower number of logical cores in order to represent more accurately the number of Workers that can run at once, so don't treat this as an absolute measurement of the number of cores in the user's system.

Therefore, what we assessed on IRC was already considered by them. Cool.